source/crud/read-operations/skip.txt

```diff
@@ -0,0 +1,58 @@
+=====================
+Skip Returned Results
+=====================
+
+.. default-domain:: mongodb
+
+
+Use skip to omit documents from the beginning of the list of returned
+documents for a read operation. You can combine skip with sort to omit
+the top (for descending order) or bottom (for ascending order) ``N``
+results for a given query. Since the order of documents returned is
+:manual:`not guaranteed
+</reference/method/cursor.sort/#result-ordering>` in the absence of a
+sort, using skip without using sort omits arbitrary documents.
+
+If the value of skip exceeds the number of matched documents for your
```

Comment:
A couple requests for this sentence:
1) Could "value of skip" be described as the parameter passed to skip instead? Otherwise, it sounds like `skip` is a variable and not a method.
2) Would it be possible to replace the second instance of a noun with a pronoun for "your query, your query"? I think "it" works ok.
3) I think there are two tenses here, but would prefer just one: "If x is y" (present perfect?) and "y will z" (future).

Reply:
1) Tricky, because depending on the method, skip can be either a field (always) or a cursor method (only sometimes). I think I'd prefer to leave it as is so the sentence applies to either usage.
2) changed to "for a query, that query" instead
3) "will return" -> "returns"